tests/nohead_test.py

```diff
@@ -0,0 +1,22 @@
+"""Tests trying to create a UDTube model without any classification heads."""
+
+import unittest
+
+from udtube import models, modules
+
+
+class NoHeadTest(unittest.TestCase):
+
+    def test_no_head_raises_error(self):
+        with self.assertRaises(modules.Error):
+            # Uses all default parameters except the heads.
+            _ = models.UDTube(
```

Comment:
do you really even need this assignment

Reply:
no but flaking tools might complain if you don't (and you want to assign to `_` or something like `unused_*` to keep them from complaining it's never used)